docs/index.rst

```diff
@@ -161,8 +161,8 @@ of Read the Docs and the larger software documentation ecosystem.
 
 * **Getting involved with Read the Docs**:
   :doc:`Contributing <contribute>` |
+  :doc:`Docker development setup </development/standards>` |
   :doc:`roadmap` |
-  :doc:`gsoc` |
```

Comment:
Was this purposefully removed? This will make that doc not listed anywhere really.

Reply:
Yea -- It should still be in the sidebar. Doesn't seem worth calling out since we're not in it this year?